### ISSUE TITLE
Same number of Raw Peaks and Naive Overlap Peaks

### ISSUE THREAD

**Issue by ajlawler** [kundajelab/atac_dnase_pipelines#68]
Hi all,

Thanks for putting this together. When I run my PE replicate samples, I am getting the same (exact) number for Raw Peaks and Naive Overlap Peaks in the ATAqc html report, and 0 IDR peaks. I have verified that I called the correct read 1 and read 2 files. I ran a different set of replicates through the pipeline and had the same problem. Is there something I might be doing wrong?

**Comment by leepc12** [comment]
Can you email those HTML reports to me (leepc12 at gmail dot com)?

**Comment by leepc12** [comment]
Yes, "Raw peaks" is identical to "Naive overlap peaks" in the ATAQC report. They are actually naive overlap peaks (not raw peaks directly called from MACS2). This naming can be a bit confusing. We will change it in the next version of the pipeline.
